### ISSUE TITLE
error running the example!

### ISSUE THREAD

**Issue by LK-Simon** [horazont/python3-openttd#1]
Installed your module (no issues there).

Go to run your watch_companies_and_clients.py demo, provide the three parameters in order (ip, port, password) and the following exception is thrown:

```
Traceback (most recent call last):
  File "watch_companies_and_clients.py", line 136, in <module>
    args.password))
  File "/usr/lib64/python3.6/asyncio/base_events.py", line 484, in run_until_complete
    return future.result()
  File "watch_companies_and_clients.py", line 39, in main
    yield from client.connect_tcp(hostname, port)
  File "/usr/local/lib/python3.6/site-packages/openttd/admin.py", line 764, in connect_tcp
    yield from self.connect(protocol, **kwargs)
NameError: name 'kwargs' is not defined
```

**Comment by horazont** [comment]
> *draws chalk lines on black tiled floor*
> *places candles*
> *sings enchantments*
> *summons eldritch gods to perform acts of necromancy on this dead repository*

In all seriousness: Thanks for reporting this. I took a look and discovered that I had a patch for that in my local working directory, but it was neither committed nor (obviously) pushed.

So I took the opportunity and committed it, along with the style fixes which also were there, updated the code to work with Python 3.7+ and released version 0.2.0. Enjoy!
